site/content/docs/latest/background/cryptography.md

```diff
@@ -1,5 +1,36 @@
 # Cryptography details of Sealed Secrets
 
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
How is this file related to this PR?

Reply:
I'm using the [doctoc](https://github.com/bitnami-labs/sealed-secrets/tree/main/docs/developer#prerequisites) generation format for the document TOC. Is that not correct?

Reply:
Just doublechecking as the PR only mentions a FAQ and not a TOC for the Cryptography document.

Reply:
Ahh true, sorry, that was not related, fixing the PR description